### PR TITLE
(12.0.0) INFRASYS-6951: Add default log handlers for salinity

### PR DESCRIPTION
**Pull request by vmadura**

### CODE REVIEW

**Comment by taoistmath** [comment]
:sake:

File: salinity/salinity/templates/index.html [hunk 1]
```diff
@@ -54,14 +54,14 @@
 {% if forloop.first %}
 <table border=1 style="display: inline-block;">
 {% endif %}
-{% ifequal result.env 'ci' %}
+{% if result.env == 'ci' or result.env == 'bottle' %}
```
Comment (adamjkeller) [inline salinity/salinity/templates/index.html]: Not sure with jinja, but I believe you could make this a bit easier to read:

```
{% if result.env in ['ci, 'bottle'] %}
```

File: salinity/salinity_front/models.py [hunk 2]
```diff
@@ -25,7 +28,7 @@ def check_failed_role(self, role, env, server_list=['aw1-web30-qa']):
         try:
             server_list = self.get_server_list(role,env)
         except Exception as e:
-            print "There was a problem, {}, using default server_list".format(e)
+            self.logger.error("There was a problem, {}, using default server_list".format(e))
```
Comment (sbraverman) [inline salinity/salinity_front/models.py]: Please add indices for formatting compatibilities e.g. {0}

Comment (adamjkeller) [inline salinity/salinity_front/models.py]: +1 ^^

**Comment by vmadura** [comment]
Made changes per suggestions. @adamjkeller @sbraverman - Please review.

**Comment by adamjkeller** [comment]
+1 looks good @vmadura

**Comment by sbraverman** [comment]
+1

**Pull request merged**